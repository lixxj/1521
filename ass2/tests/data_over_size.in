a = malloc 1000
b = malloc 2000
c = malloc 1900
free b
d = malloc 1000
e = malloc 900
f = malloc 85