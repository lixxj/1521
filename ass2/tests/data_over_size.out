test3: couldn't allocate 85 bytes for `f'
+00000 (F, 5000) 
[a] +00008 
+00000 (A, 1008) +01008 (F, 3992) 
[a] +00008 [b] +01016 
+00000 (A, 1008) +01008 (A, 2008) +03016 (F, 1984) 
[a] +00008 [b] +01016 [c] +03024 
+00000 (A, 1008) +01008 (A, 2008) +03016 (A, 1908) +04924 (F,   76) 
[a] +00008 [c] +03024 
+00000 (A, 1008) +01008 (F, 2008) +03016 (A, 1908) +04924 (F,   76) 
[a] +00008 [c] +03024 [d] +01016 
+00000 (A, 1008) +01008 (A, 1008) +02016 (F, 1000) +03016 (A, 1908) +04924 (F,   76)
[a] +00008 [c] +03024 [d] +01016 [e] +02024 
+00000 (A, 1008) +01008 (A, 1008) +02016 (A,  908) +02924 (F,   92) +03016 (A, 1908)
+04924 (F,   76) 
